stockflux-watchlist/src/services/formatters.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+
+export const truncate = input => {
```

Comment:
Just a note on this. I noticed that the current frontend apps were stripping down stock names themselves by removing everything from the first occurrence of `(`. This is because the ticker symbol was included in the name e.g. *Apple Inc. (AAPL)* in the data provided by Quandl. Going forward this should no longer be necessary, as the backend will take care of it, since other souces don't include the ticker in the name (like IEX Trading). For now I think it's alright to leave this in but when we start connecting the frontend apps to the new backend, this can be removed.